abletonosc/clip.py

```diff
@@ -70,27 +70,49 @@ def clip_callback(params: Tuple[Any]) -> Tuple:
             "remove_notes_by_id"
         ]
         properties_r = [
+            ##"end_time" ## doesn't seem to do anything, can't test
             "file_path",
             "gain_display_string",
+            ##"groove", ## if other than None, says "Error handling OSC message: Infered arg_value type is not supported"
```

Comment:
This is because the Groove type doesnt return a simple float/string, but needs handling separately in the AbletonOSC code. Can you collect all of the properties that give "arg_value type is not supported", and move them to a separate comment block beneath this list as a to-do list to tackle later?